common/config/rush/.pnpmfile.cjs

```diff
@@ -26,13 +26,21 @@ module.exports = {
  * The `context` parameter provides a log() function.
  * The return value is the updated object.
  */
-function readPackage(packageJson, context) {
+function readPackage(pkg, context) {
 
   // // The karma types have a missing dependency on typings from the log4js package.
-  // if (packageJson.name === '@types/karma') {
+  // if (pkg.name === '@types/karma') {
   //  context.log('Fixed up dependencies for @types/karma');
-  //  packageJson.dependencies['log4js'] = '0.6.38';
+  //  pkg.dependencies['log4js'] = '0.6.38';
   // }
 
-  return packageJson;
+  // Installation script of ws 0.4.32 does some weird things as the result other
+  // packages could be affected by its output, specifically @babel/types
+  // This ws version is kurento-client dependency
+  if (pkg.name === 'websocket-stream' && pkg.version.startsWith('0.')) {
+    context.log('Fix outdated ws dependency version')
+    pkg.dependencies.ws = '1.1.5';
+  }
+
```

Comment:
Why this file is different? Different `rush`/`pnpm` versions?

Reply:
Do I understand correctly that we have `rush` over `pnpm`? In general what we do here is changing internal dependency of `websocket-stream` on the fly, this happens during `rush update`.